esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -194,7 +205,7 @@ WiFiClient client;
 BluetoothSerial SerialBT;
 #endif
 
-#elif (WIRELESS_PROTOCOL == 2) // WiFi UDPCl
+#else // (WIRELESS_PROTOCOL == 2) WiFi UDPCl
```

Comment:
why this change?

Reply:
Because the matching #if is "#if (WIRELESS_PROTOCOL != 2) // not UDPCl" and there is no point in #elif with the same condition tested again.

Reply:
yes, but it aligns better with all the other ifdefs
I would wish changes would have been made with the philosophie to change as little as possible to the esp32 code :)